nym-data-observatory/src/chain_scraper/mod.rs

```diff
@@ -0,0 +1,67 @@
+use crate::db::DbPool;
+use crate::env::vars::{
+    NYXD_SCRAPER_START_HEIGHT, NYXD_SCRAPER_UNSAFE_NUKE_DB,
+    NYXD_SCRAPER_USE_BEST_EFFORT_START_HEIGHT,
+};
+use nyxd_scraper_psql::{PostgresNyxdScraper, PruningOptions};
+use std::fs;
+use tracing::{info, warn};
+
+pub(crate) mod webhook;
+
+pub(crate) async fn run_chain_scraper(
+    config: &crate::config::Config,
+    connection_pool: DbPool,
+) -> anyhow::Result<PostgresNyxdScraper> {
+    let websocket_url = std::env::var("NYXD_WS").expect("NYXD_WS not defined");
```

Comment:
rather than explicitly trying to read from env here, could we perhaps pass those values ass part of config or cli args (and then let clap handle env for us)?